models/companyOpportunity.js

```diff
@@ -0,0 +1,48 @@
+var Sequelize = require("sequelize");
+var sequelize = require("./db");
+
+var CompanyOpportunity = sequelize.define('company_opportunity', {
+    /**
+     * Title of the opportunity.
+     */
+    title: Sequelize.STRING,
+
+    /**
+     * Name of the company.
+     */
+    companyName: Sequelize.STRING,
+
+    /**
+     * Description of the opportunity.
+     */
+    description: Sequelize.STRING,
+
+    /**
+     * Link to the image of the company.
+     */
+    imageUrl: Sequelize.STRING,
+
+    /**
+     * Email to contact if you want the opportunity.
+     */
+    contactEmail: Sequelize.STRING,
+
+    /**
+     * Link to the opportunity on the website of the company.
+     */
+    link: Sequelize.STRING,
+
+    /**
+     * Level of education needed for the opportunity
+     */
+    educationLevel: Sequelize.STRING,
+
+    /**
+     * Category of the opportunity
+     */
+    category: Sequelize.STRING
```

Comment:
Should we enforce some restrictions here? As in make a enum or something?

Reply:
yeah but we should check that later anyway cuz that has to happen for the whole database etc